### ISSUE TITLE
Fail to understand what is the docker's doing 

### ISSUE THREAD

**Issue by julesbelansolidstateofmind** [RobotecAI/ros2-for-unity#69]
Reading both the docker readme and the issue, I don't understand what is the docker doing, does it only allow user to build custom message on a docker or does it virtualise ros-for-unity, making it possible to use it on any os with docker installed?

A couple of lines in the docker readme to explain its purpose in layman's terms would go a long way for beginner with the library

**Comment by pijaro** [comment]
Thank you for the feedback, we will look into the docker scripts and make it more clear how to use them :+1:
